### ISSUE TITLE
Add search bar to messages

### ISSUE THREAD

**Issue by jgeaso1266** [swent-sp-2024-event-radar/event-radar-app#263]
## Acceptance Criteria
- use the reusable search bar in the Messages Overview Screen
- the search bar should be able to search over the names of the friends
  - one possible extra extension is to be able to search among messages (not a priority)

**Comment by jgeaso1266** [comment]
Blocked by #88

**Comment by jgeaso1266** [comment]
Completed in #200
